src/styles/BaseStyles/src/button.module.css

```diff
@@ -0,0 +1,29 @@
+:root {
+  --denhaag-button-border-radius: 3px;
+  --denhaag-button-border-width: 0.0625rem;
+  --denhaag-button-focus-border-color: var(--denhaag-ocher-4);
+  --denhaag-button-focus-border-width: var(--denhaag-focus-border-width);
+  --denhaag-button-font-family: var(--denhaag-font-family);
+
+  --denhaag-button-primary-action-background-color: var(--denhaag-green-3);
+  --denhaag-button-primary-action-color: var(--denhaag-white);
+  --denhaag-button-primary-action-hover-background-color: var(--denhaag-green-4);
+  --denhaag-button-primary-action-hover-color: var(--denhaag-white);
+  --denhaag-button-primary-action-disabled-background-color: var(--denhaag-grey-2);
+  --denhaag-button-primary-action-disabled-color: var(--denhaag-white);
+
+  --denhaag-button-secondary-action-background-color: var(--denhaag-white);
+  --denhaag-button-secondary-action-color: var(--denhaag-green-3);
+  --denhaag-button-secondary-action-border-color: var(--denhaag-green-3);
+  --denhaag-button-secondary-action-hover-background-color: var(--denhaag-white);
+  --denhaag-button-secondary-action-hover-color: var(--denhaag-green-4);
+  --denhaag-button-secondary-action-hover-border-color: var(--denhaag-green-4);
+  --denhaag-button-secondary-action-disabled-background-color: var(--denhaag-white);
+  --denhaag-button-secondary-action-disabled-color: var(--denhaag-grey-2);
+  --denhaag-button-secondary-action-disabled-border-color: var(--denhaag-grey-2);
+
+  --denhaag-button-medium-size-padding-block: 8px;
+  --denhaag-button-medium-size-padding-inline: 16px;
+  --denhaag-button-large-size-padding-block: 10px;
+  --denhaag-button-large-size-padding-inline: 20px;
```

Comment:
Some of these tokens have a value that refer to a design system token, and some that refer to a very The Hague brand specific token (such as `--denhaag-ocher-4`). What would your advice be for this? @Robbert 

I've split the brand tokens into a specific package, which means that for other developers, these tokens have no value. Is that sufficient? I can think of a few ways to approach this:
- Do not fill or define the button design tokens at all. This means that the external developer _must_ define all the tokens before they can use a component.
- Do not fill the button design tokens if it's not essential styling. This means that the padding tokens will continue to have a value, but most color tokens will not be defined.
- Define all the tokens, giving them a generic value (such as grey tones). Although it's clean to read and might allow for a non-branded version of the storybook, it has pretty much no value to the users, because they won't be using a grey component library.
- Keep this implementation, where some tokens values are brand specific, but the tokens they refer to don't exist if the user isn't importing the brand package.

Reply:
Indeed, for colors I don't feel it is feasible to provide a usable fallback value right here in the component. It is possible to pick a limited set of common design tokens for meaningful colors, that you consistently use as fallback value. That way it is easy to quickly apply colors from your brand color palette to these meanings, and quickly achieve a consistent styling.

This page contains a couple of examples for inspiration:
 https://designsystem.wesaynih.com/design-token/color/

Examples:
- success / error
- valid / invalid
- danger / warning / safe
- positive amount / negative amount
- primary action / secondary action / disabled action / destructive action / publication action
- separator color
- text color
- non-essential text color

Reply:
Will be implementing meaningful color tokens. I think they would belong in the common tokens package